swift/IceDiscovery/helloUI/views/ContentView.swift

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+import Darwin
+import Ice
+import PromiseKit
+import SwiftUI
+
+struct ContentView: View {
+    func loadCommunicator() -> Ice.Communicator {
```

Comment:
I like that this logic was moved into it's own function, but I think it belongs in Client, instead of ContentView.
I'm guessing ContentView is like the 'main' view?